src/Components/Web/src/Forms/EditForm.cs

```diff
@@ -17,6 +17,7 @@ public class EditForm : ComponentBase
         private readonly Func<Task> _handleSubmitDelegate; // Cache to avoid per-render allocations
 
         private EditContext? _fixedEditContext;
+        private EditContext? _providedEditContext;
```

Comment:
I'm not super familiar with this area, do we need the extra field? Or can we make _providedContext hold the right value whether it received a Model or a Context

Reply:
The `_providedEditContext` field is there to track if the `EditContext` was provided directly via the `EditContext` parameter. This mean we don't break the current functionality in the `OnParametersSet` method which checks if the developer has provided a value for both `EditContext` and `Model`.